### DISCUSSION TITLE
Developing themes with a hot-reload dev server.

### DISCUSSION THREAD

**Discussion by Tackoil** [vuepress/core#988] [Q&A] [answered]
Is there some way to developing theme by watching the update of theme files immediately? 
I tried to develop a theme from sratch. But it seems that I need to close the server and restart it every time, when I made a change in a theme file. So it'll be nice if it could watching the change (or have a opotion only in developing themes) and reload it like the behavior of the markdown files changed.

**Comment by meteorlxy** [comment, marked as answer]
It should be supported out of the box. You could provide a reproduction if you think it's a bug.

**Reply by Tackoil** [reply]
Thank you for your reply. I tried my best to make a reproduction but failed. So I decied to start from a clean one. If I meet the same error and can be reproducted, I'll open a issue about it.
